client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss

```diff
@@ -42,30 +42,6 @@
 		}
 	}
 
-	&.link-in-bio {
-		background-color: #d0cce3;
-
-		.step-container {
-			background-image:
-				url(calypso/assets/images/onboarding/link-in-bio-intro-left.webp),
-				url(calypso/assets/images/onboarding/link-in-bio-intro-right.webp);
```

Comment:
We should probably delete these images.

Reply:
Good catch, removed on 8d78ba587de54f3ded6a3cce2c285049530e45d4